docs/descriptors/xbe2_unknown.md

```diff
@@ -0,0 +1,690 @@
+00 25 64 75 08 95 01 b1 02 c0 06 00 ff 09 03 a1 02 85 08 09
+08 15 00 25 64 75 08 95 01 b1 02 09 09 15 00 25 64 75 08 95
+01 b1 02 09 0a 15 00 26 ff 00 75 08 95 01 b1 02 c0 06 00 ff
+09 04 a1 01 85 09 09 0b 15 00 25 64 75 08 95 01 b1 02 09 0c
+15 00 25 64 75 08 95 01 b1 02 09 0d 15 00 25 64 75 08 95 01
+b1 02 09 0e 15 00 26 ff 00 75 08 95 01 b1 02 09 0f 26 ff 00
+75 08 95 3c b2 02 01 c0 06 00 ff 09 05 a1 01 85 0a 09 10 15
+00 27 ff ff ff 7f 75 20 95 01 81 02 09 11 15 00 27 ff ff ff
+7f 75 20 95 01 81 02 09 12 15 00 26 ff 00 75 08 95 02 81 02
+09 13 15 00 26 ff 00 75 08 95 01 81 02 c0 06 00 ff 09 06 a1
+02 85 0b 09 14 15 00 25 64 75 08 95 01 b1 02 c0 c0 05 01 09
+06 a1 01 85 05 05 07 19 e0 29 e7 15 00 25 01 75 01 95 08 81
+02 95 01 75 08 81 03 95 06 75 08 15 00 25 65 05 07 19 00 29
+65 81 00 c0 00
+```
+
+## Packet structure (as captured by `btmon`)
+
+```
+a1 01 LX LX LY LY RX RX RY RY LT LT RT RT DP BB  
+ST SE LX LX LY LY RX RX RY RY LT LT RT RT DP BB // Copy of the first row (except first two bytes)
+ST SE PP 00 PR TS 00 00 00 00 00 00 00 00 00 00 // First two bytes are a copy of the first two bytes from above
+00 00 00 00 00 00 00 00   
+```
+        
+LX, LY, RX, RY: X and Y axes for the left and right sticks (00 00 - ff ff)
+        
+LT, RT: Going from 00 00 to FF 03
+        
+DP: D-Pad info
+0 -> nothing
+01-08 -> N, NE, E, SE, S, SW, W, NW
+        
+BB: on/off for ABXY, LB, RB
+0bRL0Y X0BA // R = RB, L = LB
+
+If Profile != 00, then ONLY THE FIRST ROW's (!!!) BB are on/off values for the paddles:
+0b0002 4013 // 1 = P1, etc.
+
+ST: on/off for Start and Sticks (thumbstick buttons)
+0b0000 SRL0 // S = Start, R/L = Right/Left thumbstick button
+
+SE: on/off for Select
+0b0000 000S // S = Select
+
+PP: on/off Paddles
+0b0000 4321 // 1 = P1, etc.
+
+PR: Profile
+00-03 -> Number of LEDs on (= Profile number)
+
+TS: Trigger settings
+0b0000 RrLl // R/L = RT/LT has 2 states, r/l = RT/LT only goes halfway.
+
+Pressing the connection button sends a new packet, but there's nothing in the packet about it
+
+
+## Parsed descriptor
+
+(via https://eleccelerator.com/usbdescreqparser/)
+
+````
+0x05, 0x01,        // Usage Page (Generic Desktop Ctrls)
+0x09, 0x05,        // Usage (Game Pad)
+0xA1, 0x01,        // Collection (Application)
+0x85, 0x01,        //   Report ID (1)
+0x09, 0x01,        //   Usage (Pointer)
+0xA1, 0x00,        //   Collection (Physical)
+0x09, 0x30,        //     Usage (X)
+0x09, 0x31,        //     Usage (Y)
+0x15, 0x00,        //     Logical Minimum (0)
+0x27, 0xFF, 0xFF, 0x00, 0x00,  //     Logical Maximum (65534)
+0x95, 0x02,        //     Report Count (2)
+0x75, 0x10,        //     Report Size (16)
+0x81, 0x02,        //     Input (Data,Var,Abs,No Wrap,Linear,Preferred State,No Null Position)
+0xC0,              //   End Collection
+0x09, 0x01,        //   Usage (Pointer)
+0xA1, 0x00,        //   Collection (Physical)
+0x09, 0x32,        //     Usage (Z)  // FIXED
```

Comment:
This is the original device descriptor, not the patched one. I'll remove my comments at the end, so please ignore these for now